gnucash/gnucash-commands.cpp

```diff
@@ -118,6 +118,12 @@ write_report_file (const char *html, const char* file)
     // ofs destructor will close the file
 }
 
+static void
+update_message_cli (const gchar *msg)
```

Comment:
Leaving aside using lambdas, this name and its gui counterpart do a poor job of communicating intent. Since the parameter they're passed to is called `update_message_cb` they would be better named `cli_update_message_cb` and `gui_update_message_cb`.